__tests__/schema/uuid/abstract-uuid.ts

```
@@ -216,6 +216,23 @@ describe('uuid', () => {
     })
   })
 
+  test('returns an error when alias contains the null character', async () => {
```

Is it the right approach to return an error instead of `null`? This would mean:

* CF: Entry would not end in the CF cache and the CF worker would use the legacy backend :heavy_check_mark: 
* Frontend: Would not display uuid and report this to sentry
* API: Would report the error to sentry

---

Sounds good. This error should only occur if both the cache value and the therefore newly fetched value are both invalid.